### ISSUE TITLE
PRM不是为每个步骤输出一个值吗？怎么输出了两个值

### ISSUE THREAD

**Issue by ccc210** [pixas/MedSSS#3]
跑了huggingface的示例代码，https://huggingface.co/pixas/MedSSS_PRM
输出的value值是[[0.55078125, -2.765625], [0.86328125, -3.328125]]，怎么输出了两个值？而且也不是0-1之间的数值啊？

**Comment by pixas** [comment]
sorry for the confusion. We have recently updated the model and the paper, and we will update the running script and corresponding models soon. 

For this issue, the two values are the logit for 0 and 1. So just perform a softmax operation and take the second value, which is just the correctness of current step

**Comment by ccc210** [comment]
> sorry for the confusion. We have recently updated the model and the paper, and we will update the running script and corresponding models soon.
> 
> For this issue, the two values are the logit for 0 and 1. So just perform a softmax operation and take the second value, which is just the correctness of current step

I get it. Thank you. I have corrected that in my code.
